swarm/src/main/resources/swarm/swarm.bom

```diff
@@ -253,7 +253,8 @@ brooklyn.catalog:
                 shell.env:
                   SWARM_ENDPOINTS:
                     $brooklyn:attributeWhenReady("swarm.endpoints")
-                command: |
+                command: |                  
+                  cd ${RUN_DIR}
```

Comment:
I have observed this command being called when we weren't in the RUN_DIR. This resulted in the servers.conf file being created in home. 

I am not 100% sure this is needed, but I think it is safer to keep it in.